targets/frontend/src/components/changes/ViewDiff.js

```diff
@@ -16,7 +17,7 @@ export const ViewDiff = ({ sx, type, inputA, inputB }) => {
   const [mode, setMode] = useState(type);
   const diff = fnMap[mode](inputA, inputB);
 
-  const groupName = useMemo(() => Math.random());
```

Comment:
not sure about this one, since it comes from cdtn-veille-git

Reply:
I had a linit warning they where saying it is useless if no second argument is used. I checked, seems legit since it rebuilds at every render without a second argument 🤷